build.sbt

```diff
@@ -407,21 +426,7 @@ lazy val codegen = projectMatrix
       "alloyVersion" -> Dependencies.Alloy.alloyVersion
     ),
     buildInfoPackage := "smithy4s.codegen",
-    libraryDependencies ++= Seq(
```

Comment:
I'm not sure I understand the rationale behind extracting this. In theory, you shouldn't have to.

Reply:
I couldn't figure out the right sbt incantation to reference the dependancies of the module directly, without extracting them into their own setting. 

This could simply be weak sbt-foo .